expected/16.0/types/macaddr8.out

```diff
@@ -0,0 +1,867 @@
+ 49 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 50 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 51 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 52 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+ 53 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+ 54 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+ 55 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+ 56 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+ 57 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+ 58 | 08:00:2f:01:02:03:04:05 | text    | 23 | 08:00:2f:01:02:03:04:05
+(58 rows)
+
+--Testcase 078:
+ALTER FOREIGN TABLE "type_MACADDR8" ALTER COLUMN "m" OPTIONS (SET column_type 'BLOB');
+--Testcase 079:
+SELECT * FROM "type_MACADDR8+" where "m" = '08:00:2b:01:02:03:04:05';
+ i  |            m            |    t    | l  |           tx            
+----+-------------------------+---------+----+-------------------------
+  1 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+  2 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08-00-2b-01-02-03-04-05
+  3 | 08:00:2b:01:02:03:04:05 | text    | 17 | 08002b01:02030405
+  4 | 08:00:2b:01:02:03:04:05 | text    | 17 | 08002b01-02030405
+  5 | 08:00:2b:01:02:03:04:05 | text    | 19 | 0800.2b01.0203.0405
+  6 | 08:00:2b:01:02:03:04:05 | text    | 19 | 0800-2b01-0203-0405
+  7 | 08:00:2b:01:02:03:04:05 | text    | 16 | 08002b0102030405
+ 16 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 17 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 18 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 19 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 20 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 21 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 22 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 23 | 08:00:2b:01:02:03:04:05 | integer | 18 | 576508035632137221
+ 31 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 32 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 33 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 34 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 35 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 36 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 37 | 08:00:2b:01:02:03:04:05 | blob    |  8 | \x08
+ 45 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 46 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 47 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 48 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 49 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 50 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+ 51 | 08:00:2b:01:02:03:04:05 | text    | 23 | 08:00:2b:01:02:03:04:05
+(29 rows)
+
```

Comment:
Could you add some test cases for macaddr or macaddr8 with operators. I would like to confirm the following specification:  
```
Operations with macaddr or macaddr8 data are not pushdowned.
```

Reply:
Added some TCs according https://www.postgresql.org/docs/17/functions-net.html

Reply:
Confirmed. Thanks. Please help me to resolve this comment.